python/README.md

```diff
@@ -36,7 +36,7 @@ response = requests.get('URL OF YOUR GTFS-REALTIME SOURCE GOES HERE')
 feed.ParseFromString(response.content)
 for entity in feed.entity:
   if entity.HasField('trip_update'):
-    print entity.trip_update
+    print (entity.trip_update)
```

Comment:
Good observation! A small detail: according to [PEP8](https://peps.python.org/pep-0008/) best practices, blank spaces between a function name and parameter list should be avoided, so I would suggest removing it before merging the PR.
```suggestion
    print(entity.trip_update)
```